libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -78,6 +79,7 @@
     private String mGoogleEmail;
     private String mRequestedEmail;
     private boolean mIsSocialLogin;
+    private boolean mIsValidEmail = true;
```

Comment:
Why set the email as true since you won't know if it is valid until it's checked. I think it's better to leave the default value of false.

Reply:
Done :D